isdi

```diff
@@ -15,12 +15,11 @@ if __name__ == "__main__":
         print("Checking mode = {}\nApp flags: {}\nSQL_DB: {}"
               .format(config.TEST, config.APP_FLAGS_FILE,
                       config.SQL_DB_PATH))
-    print("TEST={}".format(config.TEST))
-    init_db(app, sa, force=config.TEST)
+    init_db(app, sa, force=False)
```

Comment:
Similarly to below, I wonder if instead we just want to use a ternary to default TEST to false if it does not exist.

Reply:
I thought about this, i.e. should we just change the default or remove it entirely? 

I went with the latter mostly because we're planning on doing quite a bit of work related to  (1) setting up a new testing framework (2) re-factoring the shell scripts into Python scripts and (3) containerizing individual runs and (4) possibly removing the database. 

i think in light of these planned changes it makes more sense to just remove the test mode hooks. but if there's significant disagreement on this, we can just change the defaults.

Reply:
I'm not so too familiar with things, but I wonder if slowly phasing things out will make it less likely to break in existing deployments...

Otherwise things LGTM!

Reply:
Okay, then let's keep the config.TEST, but change `TEST=0` by default.